state/session/operations/completeSession.js

```diff
@@ -0,0 +1,18 @@
+import { createLogic } from 'redux-logic';
+import cookie from 'cookie_js';
+
+import { COMPLETE_SESSION } from '../types';
+import { completeSession } from '../actions';
+
+const requestSessionLogic = createLogic({
+  type: COMPLETE_SESSION,
+  warnTimeout: 0,
+
+  process(dispatch, done) {
```

Comment:
Does it work? Because according this doc https://github.com/jeffbski/redux-logic/blob/master/docs/api.md#createlogic process function requires object as first argument and dispatch as second.

Reply:
You are right. According to the documentation, depObj is necessary.
But I didn't have any error in my console when I skip depObj.
I am trying to add {} as the first parameter and my application goes down.
I'll show you tomorrow on the daily.

Reply:
Fixed completing session.